web/src/upload.js

```diff
@@ -235,15 +235,14 @@ export function setup_upload(config) {
             },
         },
     });
-    uppy.setMeta({
-        csrfmiddlewaretoken: csrf_token,
-    });
-    uppy.use(XHRUpload, {
-        endpoint: "/json/user_uploads",
+    uppy.use(Tus, {
+        endpoint: "/chunk-upload",
         formData: true,
         fieldName: "file",
         // Number of concurrent uploads
         limit: 5,
+        retryDelays: [0, 1000, 3000, 5000],
+        chunkSize: 25000000,
```

Comment:
Using 25MB chunk size, later we can get this from realm setting.

Reply:
RetryDelays indicates time after which Tus will retry to upload if any request fails in between.

Reply:
Can you add a comment linking to the documentation for `retryDelays` and any notes on how you picked those values?